src/routes/health.ts

```diff
@@ -0,0 +1,14 @@
+import { Router, Request, Response } from 'express';
+import mongoose from 'mongoose';
+import HealthCheckController from '../controllers/HealthCheckController';
```

Comment:
We can use absolute paths in TS

Reply:
Add tsconfig configuration to use it, and changed paths to absolute where was necessary.

Reply:
It's not working for me correctly.